dependencies.props

```diff
@@ -84,6 +84,14 @@
       <Version>1.0.0-prerelease-00704-03</Version>
     </StaticDependency>
 
+    <!--
+      The NETNative targeting pack is in the projectn-tfs build-info, but auto-upgrade isn't wanted.
+      Verify it at a static version rather than referencing the build-info.
+    -->
+    <StaticDependency Include="Microsoft.TargetingPack.Private.NETNative">
+      <Version>1.0.1-beta-24430-00</Version>
```

Comment:
Yeah, that work is done and https://github.com/dotnet/corefx/pull/11269 on dev/api auto-upgrades to the (single) latest version of the combined package.